### PR TITLE
Added readme for user guide

### PR DESCRIPTION
**Pull request by Gerwoud**
The user guide doesn't contain a readme atm, it contains a simple one after the introduction of this PR

### CODE REVIEW

File: documentation/documentation/README.md [hunk 1]
```diff
@@ -1,41 +1,27 @@
...
-
-```
-$ GIT_USER=<Your GitHub username> yarn deploy
-```
-
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+# UGent-3 project peristerónas user guide
+
+## Introduction
+Project peristerónas has a lot of features, therefor a detailed user guide is available to consult.
```
Comment (Vucis) [inline documentation/documentation/README.md:4]: therefor -> therefore

File: documentation/documentation/README.md [hunk 2]
```diff
@@ -1,41 +1,27 @@
...
+    npm run start -- --locale [language]
+  ```
+
+  ### Deployment
+  When you're ready to deploy run the following command to run the proper version of the site:
+  ```sh
+  npm run build
+  ```
+
+  A static version will be build that you can acces in the directory `build/`, you can then run the static site by using the command:
```
Comment (Vucis) [inline documentation/documentation/README.md:23]: build -> built
acces -> access

File: documentation/documentation/README.md [hunk 3]
```diff
@@ -1,41 +1,27 @@
...
+  ```sh
+   npm run start
+   ```
+This creates a lightweight version of the site, if you want to test a certain language version of the site you can use the command:
+  ```sh
+    npm run start -- --locale [language]
+  ```
+
+  ### Deployment
+  When you're ready to deploy run the following command to run the proper version of the site:
```
Comment (Vucis) [inline documentation/documentation/README.md:18]: ...deploy run ... -> ...deploy, run ...

**Review by Vucis — CHANGES_REQUESTED** [review]

File: documentation/documentation/README.md (hunk 3 above)
Comment (Gerwoud) [reply documentation/documentation/README.md]: https://github.com/SELab-2/UGent-3/pull/330/commits/47149f65336f985a68ab6286295ab44e83de8584

File: documentation/documentation/README.md (hunk 2 above)
Comment (Gerwoud) [reply documentation/documentation/README.md]: https://github.com/SELab-2/UGent-3/pull/330/commits/47149f65336f985a68ab6286295ab44e83de8584

File: documentation/documentation/README.md (hunk 1 above)
Comment (Gerwoud) [reply documentation/documentation/README.md]: https://github.com/SELab-2/UGent-3/pull/330/commits/47149f65336f985a68ab6286295ab44e83de8584

**Review by Vucis — APPROVED** [review]

**Pull request merged**